src/pages/workspace/expensifyCard/WorkspaceExpensifyCardDetailsPage.tsx

```diff
@@ -94,6 +95,10 @@ function WorkspaceExpensifyCardDetailsPage({route}: WorkspaceExpensifyCardDetail
         });
     };
 
+    if (card?.isLoading && !card?.cardID) {
+        return <FullScreenLoadingIndicator />;
+    }
+
     if (!card && !isLoadingOnyxValue(cardsListResult)) {
```

Comment:
Could you please also update this condition? It’s potentially prone to failure because card is always an object with the loading state handling was updated in [this change](https://github.com/Expensify/App/pull/65884/files#diff-ae962772c4a3418dc70235b3715b5c5612065b8a9371dca5043b401b5cd83781R847-R849).

```suggestion
    if (!card?.cardID && !isLoadingOnyxValue(cardsListResult)) {
```